cmake/kokkoskernels_tpls.cmake

```diff
@@ -469,6 +470,8 @@ KOKKOSKERNELS_ADD_TPL_OPTION(ROCBLAS   ${ROCBLAS_DEFAULT}     "Whether to enable
   DEFAULT_DOCSTRING "ON if HIP-enabled Kokkos, otherwise OFF")
 KOKKOSKERNELS_ADD_TPL_OPTION(ROCSPARSE ${ROCSPARSE_DEFAULT}   "Whether to enable ROCSPARSE"
   DEFAULT_DOCSTRING "ON if HIP-enabled Kokkos, otherwise OFF")
+KOKKOSKERNELS_ADD_TPL_OPTION(ROCSOLVER ${ROCSOLVER_DEFAULT}   "Whether to enable ROCSOLVER"
+  DEFAULT_DOCSTRING "ON if HIP-enabled Kokkos, otherwise OFF")
```

Comment:
Are these docstrings correct? It looks like the default is actually OFF, even if HIP is enabled.

Reply:
It looked odd to me when I added it and forgot to circle back to it.
I think it is off by default and only on if explicitly requested.
Also I need to add some logic to ensure that rocBLAS is enabled otherwise rocSOLVER will not compile correctly...